deployments/apps/main.tf

```diff
@@ -0,0 +1,83 @@
+terraform {
+  required_version = ">= 1.0.0"
+
+  required_providers {
+    digitalocean = {
+      source  = "digitalocean/digitalocean"
+      version = ">= 2.0.0"
+    }
+  }
+}
+
+provider "digitalocean" {}
+
+resource "digitalocean_app" "server-api" {
+  spec {
+    name = "server-api"
+    region = "blr"
+
+    domain {
+      name = "api.asyncapi.com"
+      type = "PRIMARY"
+    }
+
+    ingress {
+      rule {
+        component {
+          name = "asyncapi-server-api"
+        }
+        match {
+          path {
+            prefix = "/"
+          }
+        }
+        cors {
+          allow_origins {
+           exact = "*"
+          }
+          allow_methods = ["GET", "POST", "PUT", "DELETE", "OPTIONS"]
+          allow_headers = ["*"]
+        }
+      }
+    }
+
+    service {
+      name = "asyncapi-server-api"
+      http_port = 80
+      health_check {
+        http_path = "/v1/help/validate"
```

Comment:
In K8s we used /. I dont know what / returns but as far as it is a 2xx response code, is valid. Can you check? Better to not use a particular path, we will be collecting metrics soon and this healthcheck will add false hits.

Reply:
But all other paths return `404`, due to which server deploy doesn't work. Should I add a health endpoint ?

Reply:
Then I dont get how k8s liveness probes were working 🤔. I guess the validate path can work for now.
Ideally we should add that health check path, but out of the scope of this issue. Feel free to open a new issue so we can follow up after.